config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-azure.yaml

```diff
@@ -0,0 +1,113 @@
+---
+periodics:
+- interval: 30m
+  name: ci-kubernetes-e2e-azure-scalability
+  cluster: eks-prow-build-cluster
+  decorate: true
+  decoration_config:
+    timeout: 8h
+  path_alias: k8s.io/perf-tests
+  tags:
+  - "perfDashPrefix: azure-100Nodes-master"
+  - "perfDashJobType: performance"
+  - "perfDashBuildsCount: 500"
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-capz-containerd-1-7-latest: "true"
+    preset-azure-community: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: perf-tests
+    base_ref: "master"
+    path_alias: "k8s.io/perf-tests"
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-azure
+    base_ref: main # TODO: prow-load template is only on main ATM.
+    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    workdir: true
+  spec:
+    serviceAccountName: azure
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240803-cf1183f2db-master
+      command:
+      - runner.sh
+      - ./scripts/ci-entrypoint.sh
+      args:
+      - bash
+      - -c
+      - >-
+        cd ${GOPATH}/src/k8s.io/perf-tests/ &&
+        ./run-e2e.sh cluster-loader2
+        --nodes=100 \
+        --prometheus-scrape-kubelets=true \
+        --prometheus-scrape-node-exporter \
+        --provider=aks \
```

Comment:
In the meantime should we open up a PR in the perf-tests repo advocating we rename the "aks" provider to "azure" to better represent that it is not an AKS-only provider config? If that change is accepted prior to this new test job we can update the PR here to point to the new provider value.

Reply:
Done, just opened [a PR](https://github.com/kubernetes/perf-tests/pull/2864)